elem/elem.gen.go

```diff
@@ -754,7 +754,7 @@ func Small(markup ...vecty.Markup) *vecty.Element {
 	return e
 }
 
-// The HTML <source> element specifies multiple media resources for either the <picture>, the <audio> or the <video> element. It is an empty element. It is commonly used to serve the same media content in multiple formats supported by different browsers.
+// Editorial review completed.
```

Comment:
It does.. good catch. It appears this is what is one the page at https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Embedded_content though.

The top of the page says this after shift+refresh:

![image](https://cloud.githubusercontent.com/assets/3173176/16360813/083243f2-3b2b-11e6-99cb-aa2fba8ebec0.png)

Perhaps it will update sometime soon, but history has told me it might be a while (on the order of days).